mantle/cmd/kola/testiso.go

```diff
@@ -298,7 +298,8 @@ func runTestIso(cmd *cobra.Command, args []string) error {
 		PxeAppendRootfs: pxeAppendRootfs,
 	}
 
-	if instInsecure {
+	// Default to ignore signing verification when running with development build
+	if instInsecure || strings.Contains(kola.CosaBuild.Meta.BuildID, ".dev.") {
```

Comment:
```suggestion
	// Default to ignore signing verification when running with development build
	// https://github.com/coreos/fedora-coreos-tracker/issues/908
	if instInsecure || strings.Contains(kola.CosaBuild.Meta.BuildID, ".dev.") {
```

Optional: add link to original issue where this was discussed.

Reply:
Thoughts on https://github.com/coreos/coreos-assembler/pull/2336#discussion_r682110884? I think it'd be good to log something in the `.dev.` case.